pkg/client/client_test.go

```diff
@@ -0,0 +1,39 @@
+package client
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+)
+
+// Must create new client or panic
+func mustClient(options ...Options) *Client {
+	client, err := NewClient(options...)
```

Comment:
no, this is wrong, should not panic, however the proper error should be returned and in that case the test should fail gracefully